Задание по девятому спринту

numpadCrusader opened the pull request.

Commit:
added mock avatar and exit icon assets

Commit:
did basic layout for profile view controller

Commit:
added tab bar controller

Commit:
wrapped ImageListViewController in nav controller

Commit:
added SingleImageViewController and route to it

Commit:
adjusted routing to chosen image

Commit:
changed presentation style for single image vc

Commit:
wrapped single image view in scroll view

Commit:
added zooming to single image view

Commit:
added share and like button in single image view

Commit:
adjusted actions for buttons in single image vc

Commit:
adjusted ui elements in storyboard

Commit:
remade profile view controller's layout in code

Commit:
adjusted scroll view's zooming and image size

Kirchberg commented on ImageFeed/Profile/ProfileViewController.swift:
Очень аккуратно и красиво получилось, класс ✨

Kirchberg reviewed (changes requested):
⚠️ Критические замечания (это нужно исправить обязательно):

1. При открытии SingleImageViewController была добавлена лишняя кнопка лайка, которой нет в макетах, необходимо ее удалить в соответствии с условием задания 6, [см. скриншот](https://ibb.co/HkhwBND).

Без исправлений этих пунктов, я не смогу принять у тебя работу. Если возникнут вопросы, обязательно задавай их Наставнику!

--- 

🍏 Рекомендации (это можно исправить по желанию):

1. При нажатии на ячейку, она выделяется серым. Выделение стоит убрать – настройка ячейки `Selection - None`

Комментарии ревью оставлены в Pull Request'е

numpadCrusader committed:
adjusted ui by following review comments

removed like button in `SingleImageViewController`; set cell selection style to `none`.

Kirchberg reviewed (approved):
Не забудь вмерджить пул-реквест в основную ветку.
Успехов в следующем задании!
